pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -409,57 +409,63 @@ quotes.
 
 ### Example of task
 
-    version: v1.0
-    name: The name of the Semaphore 2.0 project
-    agent:
-      machine:
-        type: e1-standard-2
-        os_image: ubuntu1804
-    blocks:
-       task:
-          jobs:
-            - name: Check environment variables
-              commands:
-                - echo $SEMAPHORE_PIPELINE_ID
-                - echo $HOME
-                - echo $PI
-                - pwd
-          prologue:
-              commands:
-                - checkout
-          env_vars:
-               - name: PI
-                 value: "3.14159"
+``` yaml
```

Comment:
When you make changes to existing pages, please use [the new convention for formatting code snippets](https://github.com/semaphoreci/docs/blob/master/CONTRIBUTING.md#text-formatting) so that syntax highlighting works.

Reply:
Update: I've now read your take on this. Looking forward to the change in file parser.